sensing/pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp

```diff
@@ -123,6 +124,8 @@ void DistortionCorrectorComponent::onPointCloud(PointCloud2::UniquePtr points_ms
     return;
   }
 
+  update_azimuth_and_distance_ = get_parameter("update_azimuth_and_distance").as_bool();
```

Comment:
Is this typo? IMO 'update_azimuth_and_distance_' is already updated in constructor so this can be remove.
```suggestion
```